dhcpv6/client6/client.go

```diff
@@ -163,16 +163,16 @@ func (c *Client) sendReceive(ifname string, packet dhcpv6.DHCPv6, expectedType d
 		if err != nil {
 			return nil, err
 		}
-		adv, err = dhcpv6.FromBytes(buf[:n])
+		adv, err = dhcpv6.MessageFromBytes(buf[:n])
```

Comment:
I think this should be `FromBytes` instead of `MessageFromBytes`.
For example in the case of using a modifier that encapsulates a message inside a relaymessage to simulate a relay. The reply will come as a relaymessage.

Reply:
Given how relays encap and decap messages, clients should only ever get raw messages, or your relays are screwed up.

Reply:
There is a valid use case where you want to simulate a relay, to do a unicast request to a particular server and bypassing the local relay. This can be done with a modifier that encapsulates the message in a relaymessage. That is why we have https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/client6/client.go#L60. The client should support receiving a relaymessage.

Reply:
And that is why we have https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/client6/client.go#L89

Reply:
I see. Included again.